splunk/scripts/index_verification.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+logs_path="$SPLUNK_HOME/test_logs"
+endpoint="https://localhost:8089/services/data/inputs/oneshot"
+
+echo "Indexing logs..."
+for file in "$logs_path"/*; do
+  echo "Indexing $file"
+  curl -k -u admin:"$SPLUNK_PASSWORD" "$endpoint" -d name="$file"
```

Comment:
Do we really need -k here? The image if missing the certificate for a a service that is running inside itself? Also prefer longer option names for better readability in scripts.

Reply:
On SSL errors?

Reply:
Yes